coral-dbt/src/main/resources/tests/setup_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Check that the correct number of arguments were provided
+if [ $# -ne 2 ]; then
+    echo "Usage: $0 source_file destination_file"
+    exit 1
+fi
+
+# Copy the file
+cp $1 $2
+
+# Remove copyright and comments (will otherwise be printed) and return call
+sed -i -e '/\/\*/,/\*\//d ; /^--/d ; s/return(\(.*\))/\1/' $2
```

Comment:
Sure, the first expression matches anything starting with `/*` up to `*/`, which should match the multi-line comment structure (used by the copyright). I'm not too concerned with this accidentally matching something else since anything with this format should be a comment. 

```
/**
 * Copyright 2023 LinkedIn Corporation. All rights reserved.
 * Licensed under the BSD-2 Clause license.
 * See LICENSE in the project root for license information.
 */
```

The second expression matches anything starting with `--`, which is the single line comment structure; again, also not too worried about this unintentionally matching something.

```
-- This is a comment
```

The third expression replaces a `return(foo)` statement with just `foo`, without loss of generality. This should not generate a mismatch either since there should be only one relevant return statement per macro. 

```
// Before
{{ return(foo) }}

// After
{{ foo }}
```